asllib/doc/Slicing.tex

```diff
@@ -187,6 +188,13 @@ \subsubsection{ASTRule.Slice\label{sec:ASTRule.Slice}}
 }
 \end{mathpar}
 
+\begin{mathpar}
```

Comment:
perhaps mention, at the beginning of this subsection, that the notation `[:w]` is syntactic sugar for `[0:w]`